awscli/customizations/s3events.py

```diff
@@ -57,6 +65,41 @@ def replace_event_stream_docs(help_command, **kwargs):
               "object content is written to the specified outfile.\n")
 
 
+def document_expires_string(help_command, **kwargs):
+    doc = help_command.doc
+    popped = []
+    current = ''
+    while current != 'Expires -> (timestamp)':
+        try:
+            current = doc.pop_write()
+            popped.append(current)
+        except IndexError:
+            # Do nothing if Expires is not in the modeled response
+            break
+    # Put back the expires field and description
+    doc.push_write(popped.pop())
```

Comment:
This requires the condition that each entry (including description) in the output syntax is made up of exactly four components on the stack. Not the cleanest solution in my opinion